tools/schema_loader.hh

```diff
@@ -19,6 +19,8 @@ namespace tools {
 /// The schema string is expected to contain everything that is needed to
 /// create the table(s): keyspace, UDTs, etc. Definitions are expected to be
 /// separated by `;`. A keyspace will be automatically generated if missing.
+/// Tables whose name ends in "_scylla_cdc_log" are interpreted as CDC tables,
+/// meaning they will be configured with the CDC partitioner.
```

Comment:
Is this true? I thought @kbr- said something about an extra check whether the table without _scylla_cdc_log also exists?
Anyway, @kbr- you're the expert here, please review.

Reply:
This check is not possible to do in this method, as the schema dumps parsed by this tool only contain the minimal amount of context for parsing the schema. This doesn't include the base table the CDC log is attached to and so this check cannot be performed. Hence the need for this heuristics and the warning comment.